src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -261,6 +261,7 @@ private static EquipmentInfos toEquipmentInfos(Line line) {
             .id(line.getId())
             .name(line.getNameOrId())
             .type("LINE")
+            .variantId("InitialState")
```

Comment:
Remove it
It's a private name in network store

Reply:
I added a variant value because, in those tests, the variants are null, which is not normal. Additionally, we need to have the value to differentiate between initialVariant and variantId

Reply:
Done